crates/types/src/simple_certificate.rs

```diff
@@ -94,22 +116,21 @@ impl<TYPES: NodeType, VOTEABLE: Voteable + Committable, THRESHOLD: Threshold<TYP
         };
         committable::RawCommitmentBuilder::new("Certificate")
             .field("data", self.data.commit())
-            .field("vote_commitment", self.vote_commitment)
```

Comment:
Can we leave this in? It would be easier, because we wouldn't have to do any version switch on the certificate